plugins/mimecast/plugin.spec.yaml

```diff
@@ -5,15 +5,15 @@ name: mimecast
 title: Mimecast
 description: Services for email security, archiving and continuity. Protect, manage
   and archive without compromise
-version: 5.3.5
+version: 5.3.6
 connection_version: 5
 supported_versions: ["Mimecast API 2022-11-07"]
 vendor: rapid7
 support: rapid7
 cloud_ready: true
 sdk:
   type: slim
-  version: 5.3.0
+  version: 5.3.2
```

Comment:
Previously tried to pin to 5.3.0 in the last release but missed the docker file so it still pulled SDK version 5.3.2